tests/filecheck/projects/riscv-backend-paper/pres.mlir

```diff
@@ -71,8 +71,8 @@ riscv_func.func @pres_4(%X : !riscv.reg<a0>, %Y : !riscv.reg<a1>, %Z : !riscv.re
     %zero_5 = riscv_snitch.scfgwi %Z, 898 : (!riscv.reg<a2>) -> !riscv.reg<zero>
     %zero_6 = riscv.csrrsi 1984, 1 : () -> !riscv.reg<zero>
     %i = riscv.addi %zero, 63 : (!riscv.reg<zero>) -> !riscv.reg<a0>
-    %x = riscv.get_float_register() : () -> !riscv.freg<ft0>
```

Comment:
it's weird that we allow this... I don't feel like fixing the whole custom syntax for riscv right now but might be good to look into later

Reply:
Not sure what you believe is weird here?

Reply:
`riscv.get_float_register()` this is an operation that cannot have operands, so allowing the `()` at the end seems off, I would expect the parser to warn the user.